rest/historical_test_data_routes_test.go

```diff
@@ -0,0 +1,117 @@
+package rest
```

Comment:
I know that the REST route mostly just devolves into parsing and calling the DB connector, but given that the parsing layer is quite involved, should we also integration test that the REST route behaves as expected when calling the DB connector (i.e. it has the same behavior as the DB connector tests as long as you pass in the right REST query parameters?)

Reply:
I thought the whole point of the mock connector was for the REST route tests. I think it would be overkill to test using the DB connector here as well.